lib/index.js

```diff
@@ -30,8 +30,16 @@ var Iterable = module.exports = integration('Iterable')
  */
 
 Iterable.prototype.track = function(payload, fn){
+  var endpoint;
+  if (payload["isUpdateCart"]) {
```

Comment:
P.S. Segment - is there a better way to change the endpoint than just set some field on the output of the mapper and switch on it?